dumpling/export/dump.go

```diff
@@ -379,6 +400,83 @@ func (d *Dumper) dumpDatabases(tctx *tcontext.Context, metaConn *sql.Conn, taskC
 	return nil
 }
 
+// adjustDatabaseCollation adjusts db collation and return new create sql and collation
+func adjustDatabaseCollation(db *sql.Conn, parser1 *parser.Parser, originSQL string, dbName string) (string, string, error) {
+	stmt, err := parser1.ParseOneStmt(originSQL, "", "")
+	if err != nil {
+		return "", "", err
```

Comment:
Not agree with returning an error here. We can still dump tables even if create database SQL can be parsed by dumpling's parser.

Reply:
ok, if there is an error,  we can add err log and contitue.